### DISCUSSION TITLE
Recipe multiplier

### DISCUSSION THREAD

**Discussion by rspierenburg** [nextcloud/cookbook#1934] [Ideas and discussions]
It would be great if we could implement an easy way to multiply a recipe on the fly.  Perhaps add a modifier parameter.

**Comment by seyfeb** [comment]
Hey! Do you mean like this #1573? This has already been implemented and should be available in the newest version of the app.

**Reply by rspierenburg** [reply]
Thats exactly what I was looking for.  This is amazing.
